src/controllers/start.controller.ts

```diff
@@ -10,7 +14,14 @@ import {
 import { Templates } from "../types/template.paths";
 
 export const get = (req: Request, res: Response) => {
+    const lang = selectLang(req.query.lang);
+    const locales = getLocalesService();
+    let currentUrl;
+    currentUrl = urlUtils.getUrlToPath(OFFICER_FILING, req)
```

Comment:
Minor: if not redeclared, could be used directly on line 24

Reply:
Valid point,  change made.